### ISSUE TITLE
[vpc-2azs] Can not access my sites

### ISSUE THREAD

**Issue by caffe** [widdix/aws-cf-templates#21]
Hello,

I have used the VPC-2azs CF Template to create my VPC and then I've launched inside of it my Opsworks Stack with my sites. I actually cloned my current stack so the new one runs inside the VPC.

The problem is that I'm not being able to access my sites. I'm using an ELB I added to my Stack. When trying to access the site's URL it keeps loading but nothing happens.

I've launched the stack in the Public subnets A and B. Do I need any additional configuration in order to have my sites accessible through the web?

I have a PHP/MySQL application. The MySQL is running on RDS.

It seems something is being blocked. Any idea what might that be?

Thank you in advance.

Best regards,
Walter

**Comment by caffe** [comment]
Hi,

I've just found out what was wrong. I had to add my instance IP in RDS Security Group. So, my App wasn't accessing the DB causing the site not to load.

I keep getting the error below in my Apache but I guess it has nothing to do with the Cloud Formation Template right?

[Sun Sep 04 22:13:51 2016] [info] Initial (No.1) HTTPS request received for child 25 (server www.mydomain.com:443)

Regards,
Walter
